pkg/controllers/monitoring/monitoring-stack/conditions_test.go

```diff
@@ -176,10 +194,68 @@ func TestUpdateConditions(t *testing.T) {
 					Reason:             PrometheusNotReconciled,
 				}},
 		},
+		{
+			name: "Prometheus observed generation is different to the Prometheus generation",
```

Comment:
(nit)
```suggestion
			name: "Prometheus observed generation is different than the Prometheus generation",
```

Reply:
(nit on nit) I think the common use is `different from` but if my understanding is right, it is fine to use `from / to / than` 

Not native speaker so take this with a large grain on salt :D

Reply:
:) https://www.deepl.com/translator tells me to use `from` :)